ansible-ipi-install/roles/bootstrap/tasks/25_copykeys.yml

```diff
@@ -13,3 +13,4 @@
     echo '{{ ansible_ssh_pass }}' | sshpass ssh-copy-id -i {{ ansible_ssh_key }} -o 'StrictHostKeyChecking no' -f {{ ssh_user }}@{{ hostname }}
   changed_when: false
   when: ssh_result.stderr_lines|length > 0
+  become: true
```

Comment:
Why is this needed, if the orchestration host is not using the `root` user, we want whatever user is on the orchestration host to copy its keys right not the root's keys?

Reply:
`sshpass` command is copying the public key from the cloned path `redhat-performance/perf-dept.git`, again airflow user couldn't ssh-copy this file so this extra privilege.

Reply:
OK should we debug why airflow user is not able to do this instead of fixing it here?

Reply:
As discussed this task copies `ansible_ssh_key` specified in `group_vars/all.yaml`. In CI, we set this to use keys from [perf-dept](https://github.com/redhat-performance/perf-dept). But anyway I tried a CI build without setting to `true` and it went thru, I am assuming the mentioned error was due to some other cause. I will revert this for now.